pkg/logs/input/docker/tailer.go

```diff
@@ -36,53 +36,55 @@ const tagsUpdatePeriod = 10 * time.Second
 // Logs from stdout and stderr are multiplexed into a single channel and needs to be demultiplexed later one.
 // To multiplex logs, docker adds a header to all logs with format '[SEV][TS] [MSG]'.
 type Tailer struct {
-	ContainerID   string
+	container     *Container
 	outputChan    chan *message.Message
 	decoder       *decoder.Decoder
 	reader        io.ReadCloser
 	cli           *client.Client
 	source        *config.LogSource
+	originSource  *config.LogSource
```

Comment:
Maybe add a comment here explaining why we have two sources.
Could be clearer if named `containerSource` or something like that